File: Python/tests/test_prl_fictitious_rp_woa.py
import pytest

from hbayesdm.models import prl_fictitious_rp_woa


def test_prl_fictitious_rp_woa():
    _ = prl_fictitious_rp_woa(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_prl_fictitious_rp_woa.R
context("Test prl_fictitious_rp_woa")
library(hBayesDM)

test_that("Test prl_fictitious_rp_woa", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(prl_fictitious_rp_woa(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
